docs/_summary.md

```diff
@@ -55,6 +55,7 @@
   * [Combos](feature_combo.md)
   * [Command](feature_command.md)
   * [Debounce API](feature_debounce_type.md)
+  * [Dip Switch](feature_dip_switch.md)
```

Comment:
Technically `Dip` should be fully capitalized as `DIP` because it's an acronym (for "dual in-line package") throughout the documentation.

```suggestion
  * [DIP Switch](feature_dip_switch.md)
```